src/helpers/purchase-operation-helper.ts

```diff
@@ -251,6 +251,30 @@ export class PurchaseOperationHelper {
           ),
         );
         return;
+      case CheckoutStatusErrorCodes.SetupIntentCompletionFailed:
```

Comment:
I would appreciate a double check on whether we need any additional handling on the SDK for this error code

Reply:
I think it’s fine. Since no action can be taken by the end customer, this seems correct.